run_demo

```diff
@@ -1,18 +1,18 @@
 #!/bin/sh
 
+export METEOR_PACKAGE_DIRS="$(dirname "$(pwd)")/"
 echo "---------------------------------"
-echo "Setting PACKAGE_DIRS to $PACKAGE_DIRS so Meteor can find this package"
+echo "Setting METEOR_PACKAGE_DIRS to $METEOR_PACKAGE_DIRS so Meteor can find this package"
 echo "---------------------------------"
-export PACKAGE_DIRS="$(dirname "$(pwd)")/"
 
-echo "---------------------------------"
-echo "Starting code linting in watch mode"
-echo "---------------------------------"
+# echo "---------------------------------"
+# echo "Starting code linting in watch mode"
```

Comment:
I wasn't a fan of this eslint watcher so I'm removing it in favor of just pre-commit git hooks